play-json/jvm/src/main/scala/play/api/libs/json/jackson/JacksonJson.scala

```diff
@@ -221,6 +244,25 @@ private[jackson] class PlaySerializers extends Serializers.Base {
 
 private[json] object JacksonJson {
 
+  /**
+   * Define limits for parsing BigDecimal numbers.
+   *
+   * By default, we are using MathContext.DECIMAL128 and then the limits are define in its terms.
+   */
+  object BigDecimalLimits {
```

Comment:
It is great to have some safe defaults, but shouldn't also be an ability to configure them for different cases?